source/game/scripts/bind_threading.cpp

```diff
@@ -398,7 +398,8 @@ struct ScriptThread {
 	}
 
 	void stop() {
-		runThread.compare_exchange_strong(1, 2);
+		std::int32_t comparand = 2;
+		runThread.compare_exchange_strong(comparand, 1);
```

Comment:
the std::atomic version of compare_exchange_strong has the arguments switched, and returns bool instead of the value currently being stored. 

The first argument is also a reference, and has it's value replaced with the value stored in the atomic object at the time the operation is initiated. Previously this would have been the return value.